apply/run.go

```diff
@@ -90,18 +88,19 @@ func (c *ClusterArgs) SetClusterArgs() error {
 		}
 		c.cluster.Spec.Network.SvcCIDR = c.svcCidr
 	}
+	if c.passwd != "" {
+		c.cluster.Spec.SSH.Passwd = c.passwd
+	}
 	if IsNumber(c.masterArgs) && (IsNumber(c.nodeArgs) || c.nodeArgs == "") {
 		c.cluster.Spec.Masters.Count = c.masterArgs
 		c.cluster.Spec.Nodes.Count = c.nodeArgs
-		c.cluster.Spec.SSH.Passwd = c.passwd
 		c.cluster.Spec.Provider = common.DefaultCloudProvider
 	} else if IsIPList(c.masterArgs) && (IsIPList(c.nodeArgs) || c.nodeArgs == "") {
 		c.cluster.Spec.Masters.IPList = strings.Split(c.masterArgs, ",")
 		if c.nodeArgs != "" {
 			c.cluster.Spec.Nodes.IPList = strings.Split(c.nodeArgs, ",")
 		}
 		c.cluster.Spec.SSH.User = c.user
```

Comment:
There is a word "failed to found Clusterfile", could you help change the "found" to "find"?